Assets/Scripts/2D/StartGuiManagerScript.cs

```diff
@@ -265,16 +286,19 @@ public void LoadSaveAction()
         _preparingWorld = true;
     }
 
+    /// <summary>Cancels the load heightmap action.</summary>
     public void CancelLoadHeightmapAction()
     {
         SetSeedDialogPanelScript.SetVisible(true);
     }
 
+    /// <summary>Cancels the load save action.</summary>
     public void CancelLoadSaveAction()
     {
         MainMenuDialogPanelScript.SetVisible(true);
     }
 
+    /// <summary>Sets the generation seed.</summary>
```

Comment:
I would change this to "opens the 'Generate New World' dialog".

As an aside, the reason the function is called 'SetGenerationSeed' is because originally that's what the only thing the dialog used to do. Since then, the dialog has grown in complexity and now it does much more than that. But this is a function that is visible on the Unity3D editor and is used directly from there. Changing it's name would break some editor links that would need to be fixed manually, which is kind of risky and that's why I haven't done it.